core/src/test/java/org/dotwebstack/framework/core/jexl/JexlHelperTest.java

```diff
@@ -141,6 +141,41 @@ public void evaluateScript_throwsException_forTypeMismatch() {
         () -> this.jexlHelper.evaluateScript("return 12;", context, String.class));
   }
 
+  @Test
+  public void evalueScriptWithFallback_fallsBack_forExceptionInScript() {
```

Comment:
```suggestion
  public void evaluateScriptWithFallback_fallsBack_forExceptionInScript() {
```
+ alle volgende testen